torch/_inductor/ir.py

```diff
@@ -1769,7 +1769,7 @@ def _dynamic_reshape_indexer(old_size, new_size):
         """
         Perform a reshape entirely by modifying indexing math
         """
-        size_hint = V.graph.sizevars.size_hint
+        size_hint = partial(V.graph.sizevars.size_hint, fallback=8192)
```

Comment:
This looks pretty sus. Can we just disable dynamic reshape indexing when things are unbacked?

Reply:
Could you please clarify what "disable" mean in this context? I.e., I see `dynamic_reshape_indexer` being used in a few places in the IR like this:

https://github.com/pytorch/pytorch/blob/main/torch/_inductor/ir.py?#L996

or here

https://github.com/pytorch/pytorch/blob/main/torch/_inductor/ir.py?#L1705

and there doesn't seem to be obvious ways to get around those calls. Or should we invoke some other (static) reshape indexer within `dynamic_reshape_indexer` if we see unbacked SymInts?

Reply:
Hmm, I am not too sure. @jansel any ideas?

Reply:
The dynamic_reshape_indexer is used to implement views.

I think we could just call `.realize()` as a fallback if there are unbacked symints.